### ISSUE TITLE
setNetwork function similar to connect

### ISSUE THREAD

**Issue by SaulDope** [aptos-labs/aptos-wallet-adapter#239]
We have our own custom wallet connector modal using the adapter.  We have an issue on our public testnet where when a user has their wallet set to mainnet, we have no way of helping them switch their wallet to testnet.  The user must manually set their wallet to testnet, then try to connect to our dapp again.

It would be great if we could show users a prompt with a button to swap networks.  This would also be useful for projects to do the opposite (ie a wallet is set to testnet but the user is trying to access the mainnet page).

There is a helper network() that shows the network.  I think it would be helpful if there was a function like setNetwork(chainId || NetworkName).

**Comment by 0xmaayan** [comment]
Agreed! Take a look at the new [wallet standard](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-62.md#specification) we are about to follow from now on, especially the  `aptos:changeNetwork` feature

**Comment by SaulDope** [comment]
That looks like the perfect fit, thanks for the response.  Looking forward to this standard.  Happy to help work on the Aptos Wallet Adapter when the AIP is finalized!
